app/client/src/components/pages/Community/config.js

```diff
@@ -307,8 +307,9 @@ const tabs = [
     icon: protectIcon,
     upper: protectUpper,
     lower: <Protect />,
-    layers: {}, // hide the nonprofits layer, re-add when nonprofits are approved
-    // layers: { nonprofitsLayer: true }
+    layers: {
+      wsioHealthIndexLayer: true,
+    },
```

Comment:
```suggestion
    layers: {
      wsioHealthIndexLayer: true,
      wildScenicRiversLayer: false,
      protectedAreasLayer: false,
    },
```

I think this should be updated to include the other layers on this tab. This makes it so users can toggle the other protect layers on/off (via the layer list) while in full screen mode.